apps/libsignpost/signbus_io_interface.h

```diff
@@ -15,23 +17,27 @@ extern "C" {
 #define SIGNBUS_DEBUG(...)
 /*
 #define SIGNBUS_DEBUG(...) do {\
-    printf("SBDBG %24s:%30s: %04d: ", __FILENAME__, __func__, __LINE__);\
-    printf(__VA_ARGS__);\
+    sprintf(port_print_buf, "SBDBG %24s:%30s: %04d: ", __FILENAME__, __func__, __LINE__);\
```

Comment:
You're not gonna like me for this, but these should really all be `snprintf`s with the size of 80 (should at least be easy to fix)